Directory.Build.props

```diff
@@ -4,7 +4,20 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <LangVersion>7.3</LangVersion>
     <TreatWarningsAsErrors Condition="'$(Configuration)' == 'Release'">true</TreatWarningsAsErrors>
-    <NoWarn>$(NoWarn);CS1591;CA1812;SA1600;SA1633;SA1200;SA1202;SA1201</NoWarn>
+    <NoWarn>$(NoWarn);CS1591;CA1812;SA1600;SA1633;SA1200;SA1202;SA1201;CA1016</NoWarn>
```

Comment:
Reconsider `CA1016`

Reply:
For now I think we should try to leave `CA1016` disabled and `GenerateAssemblyVersionAttribute` set to `false` due to similar issues described here: https://github.com/dotnet/sdk/issues/1450